assets/js/modules/ads/index.js

```diff
@@ -141,14 +141,19 @@ export const ADS_NOTIFICATIONS = {
 			// isWooCommerceActivated, isGoogleForWooCommerceActivated and isGoogleForWooCommerceLinked are all relying
 			// on the data being resolved in getModuleData() selector.
 			await resolveSelect( MODULES_ADS ).getModuleData();
+			await resolveSelect( CORE_MODULES ).isModuleConnected( 'ads' );
```

Comment:
Please, use `await Promise.all( ... )` to resolve both selectors simultaneously.